pandas/tests/sparse/frame/test_frame.py

```diff
@@ -199,6 +199,31 @@ def test_constructor_from_series(self):
         # without sparse value raises error
         # df2 = SparseDataFrame([x2_sparse, y])
 
+    def test_constructor_from_dense_series(self):
+        # getting the columns from the Series name
+        x = Series(np.random.randn(10000), name='a')
+        assert isinstance(x, Series)
+        df = SparseDataFrame(x)
+        assert isinstance(df, SparseDataFrame)
```

Comment:
construct an expected SDF and compare

Reply:
construct a DataFrame and use .to_sparse() to construct an expected frame. we do not want to do all of these little checks, we already have well established comparison functions, e.g. ``tm.assert_sparse_equal`` for this